src/pages/users/AddUsers.vue

```diff
@@ -801,9 +709,50 @@ async function submitUsers() {
         return processedUser;
       });
 
-      // This is the most likely place for an error, due to
-      // permissions, etc. If so, drop to Catch block
-      const res = await createUsers(processedUsers);
+      const submitValidationErrors = [];
+      processedUsers.forEach((processedUser, idx) => {
+        const validation = validateAddUsersSubmit(processedUser);
```

Comment:
I don't really see a point in validating something that we've just programmatically done above. There's no way it can fail. So checking it is redundant.